playground/mocks/idp/idx/data/select-factor-authenticate.json

```diff
@@ -27,6 +27,10 @@
                             {
                                 "label": "Email Label",
                                 "value": "emf2j1ccd6CF4IWFY0g3"
+                            },
+                            {
+                                "label": "Security Key or built-in authenticator",
+                                "value": "webfidocd6CF4IWFY0g5"
```

Comment:
for webauthn we can have multiple factor instances that map to  same profile and we would want to just show one entry in the UI. API changes needed.